app/models/conversation.rb

```diff
@@ -86,7 +86,7 @@ def subscribers(user)
   end
 
   def receive(user, person)
-    cnv = Conversation.find_or_create_by!(self.attributes)
+    cnv = Conversation.create_with(self.attributes).find_or_create_by!(self.attributes[:guid])
```

Comment:
Just checked, it generates the following query:

```sql
SELECT  "conversations".* FROM "conversations"  LIMIT 1
```

That just happens to match in the test.